### PR TITLE
[improve] add filename check for in home ci

### PR DESCRIPTION
**Pull request by LiuTianyou**
## What's changed?

1. added ci for filename checking following documentation contribution guidelines.
2. modify file name
3. format table in .md file

## Checklist

- [x]  I have read the [Contributing Guide](https://hertzbeat.apache.org/docs/community/code_style_and_quality_guide)
- [ ]  I have written the necessary doc or comment.
- [ ]  I have added the necessary unit tests and all cases have passed.

## Add or update API

- [ ] I have added the necessary [e2e tests](https://github.com/apache/hertzbeat/tree/master/e2e) and all cases have passed.

### CODE REVIEW

File: home/docs/help/api.md [hunk 1]
```diff
@@ -9,23 +9,23 @@ keywords: [open source monitoring tool, monitoring http api]
...
-| Username      | User name used for interface Basic authentication or Digest authentication |
-| Password        | Password used for interface Basic authentication or Digest authentication |
-| Headers        | HTTP request headers                                          |
-| Params        | HTTP query params, support [time expression](time_expression) |
-| Content-Type  | Set the resource type when carrying the BODY request body data request |
-| Request BODY    | Set the carry BODY request body data, which is valid when PUT or POST request method is used |
-| Collection interval   | Interval time of monitor periodic data collection, unit: second, and the minimum interval that can be set is 30 seconds |
+| Parameter name      | Parameter help description                                                                                                                                                |
+| ----------- |---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```
Comment (crossoverJie) [inline home/docs/help/api.md:13]: Please adjust the format of the table, you can use intellij ide to reformat the table.

**Comment by tomsun28** [comment]
hi, i found a problem during testing, that is, the default link of our monitoring help document is https://hertzbeat.apache.org/docs/help/ + app type name, in monitoring template yml. eg: dynamic_tp help doc link is https://hertzbeat.apache.org/docs/help/dynamic_tp/ , so here if we change the doc file name, the help doc link will 404

**Comment by tomsun28** [comment]
> 1. added ci for filename checking following documentation contribution guidelines.

so sorry i made a mistake for the confusion. Can we update the contribution guidelines and support this file name format?

**Comment by LiuTianyou** [comment]
> > 1. added ci for filename checking following documentation contribution guidelines.
> 
> so sorry i made a mistake for the confusion. Can we update the contribution guidelines and support this file name format?

@tomsun28  That's a good idea, it will result in fewer changes and be more secure. I will close this PR and create a new one after making the modifications. Thank you for your suggestion.

When I was making the changes, I was also concerned that it might affect users viewing the help documentation in previous versions. After my modifications, I tested it and confirmed that if the ID is defined in the document, the document link is not affected by the filename. I apologize for not mentioning this in the PR description.

**Pull request closed without merging**